model/versioning/Versioned.php

```diff
@@ -780,6 +805,95 @@ public function onAfterSkippedWrite() {
 	}
 
 	/**
+	 * Find all objects owned by the current object.
+	 * Note that objects will only be searched in the same stage as the given record.
+	 *
+	 * @param bool $recursive True if recursive
+	 * @param ArrayList $list Optional list to add items to
+	 * @return ArrayList list of objects
+	 */
+	public function findOwned($recursive = true, $list = null)
+	{
+		// Find objects in these relationships
+		return $this->findRelatedObjects('owns', $recursive, $list);
+	}
+
+	/**
+	 * Find objects which own this object.
+	 * Note that objects will only be searched in the same stage as the given record.
+	 *
+	 * @param bool $recursive True if recursive
+	 * @param ArrayList $list Optional list to add items to
+	 * @return ArrayList list of objects
+	 */
+	public function findOwners($recursive = true, $list = null)
+	{
+		// Find objects in these relationships
+		return $this->findRelatedObjects('owned_by', $recursive, $list);
+	}
+
+	/**
+	 * Find objects in the given relationships, merging them into the given list
+	 *
+	 * @param array $source Config property to extract relationships from
+	 * @param bool $recursive True if recursive
+	 * @param ArrayList $list Optional list to add items to
+	 * @return ArrayList The list
+	 */
+	public function findRelatedObjects($source, $recursive = true, $list = null)
+	{
+		if (!$list) {
+			$list = new ArrayList();
+		}
+
+		// Skip search for unsaved records
+		if(!$this->owner->isInDB()) {
+			return $list;
+		}
+
+		$relationships = $this->owner->config()->{$source};
```

Comment:
The RFC states:

> Ownership, unlike other relationship types, is only declared on the owner object via the configuration system. I.e. owns not owned_by is set. Owned by is implicit by any many_many, has_many, blongs_to, has_one, or belongs_many_many where the other side declared this object as owned

Does that mean you need to do more work here for `$source == 'owned_by'`?